core/src/main/res/values-v21/themes.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
If @macgills and @kelson42 agree that this is a good idea, you do not have to make a separate theme for v21 because of #2284 which will make v21 the minimal supported version. Maybe branch off of that branch to remove the warnings?

Reply:
@Frans-Lukas Okay if v21 becomes the minimum supported version then I'll fix it again

Reply:
Then this is blocked until the merge of that. It is definitely going ahead